controller/reviewController.js

```diff
@@ -23,8 +24,36 @@ import authenticate from '../middlewares/authMiddlewares.js';
   }
 };
 
+
+/**
+ * Retrieves the reviews of the place with the given placeId.
+ */
+const getAllReviewsByPlace = async (req, res) => {
+  try {
+    //   authenticate(req, res, next);
+    const response = await getAllReviewsByPlaceHandler(req.params.placeId);
+    res.status(response.status || StatusCodes.OK).json(response);
+  } catch (err) {
+    res.status(StatusCodes.UNAUTHORIZED).json({ message: err.message || 'User not authenticated' });
+  }
+}
+
+/**
+ * Retrieves the reviews of the user with the given username.
+ */
+const getAllReviewsByUser = async (req, res) => {
+  try {
+    //   authenticate(req, res, next);
+    const response = await getAllReviewsByUserHandler(req.params.username);
+    res.status(response.status || StatusCodes.OK).json(response);
+  } catch (err) {
+    res.status(StatusCodes.UNAUTHORIZED).json({ message: err.message || 'User not authenticated' });
```

Comment:
Send StatusCodes.INTERNAL_SERVER_ERROR here. If the user is not authenticated, we are already sending error 401 in the authenticate middleware.